### PR TITLE
Fixed issues commented in git version3 

### PR DESCRIPTION
**Pull request by isurusuranga**

### CODE REVIEW

File: modules/integration/tests-integration/bpel/src/test/java/org/wso2/bps/integration/tests/bpel/security/SecurityWithServiceDescriptorTest.java [hunk 1]
```diff
@@ -78,42 +77,46 @@ public void setEnvironment() throws Exception {
...
 
 		System.setProperty("javax.net.ssl.trustStore", trustStore);
 		System.setProperty("javax.net.ssl.trustStorePassword", "wso2carbon");
 
-		System.out.println(CarbonUtils.getCarbonHome());
-		ConfigurationContext ctx = ConfigurationContextFactory.createConfigurationContextFromFileSystem(
-								   CarbonUtils.getCarbonHome() + File.separator + "repository" +
-								   File.separator + "deployment" + File.separator + "client", null);
+		if (log.isDebugEnabled()) {
+			log.debug(CarbonUtils.getCarbonHome());
```
Comment (chrishantha) [inline modules/integration/tests-integration/bpel/src/test/java/org/wso2/bps/integration/tests/bpel/security/SecurityWithServiceDescriptorTest.java]: Instead directly logging, include some meaningful context to the log message. For example log.debug("Carbon Home: " + CarbonUtils.getCarbonHome());

File: modules/samples/product/process-tools/src/main/java/org/wso2/bps/samples/processcleanup/DBQuery.java [hunk 2]
```diff
@@ -48,12 +52,11 @@
...
-			String configPath =
-					bpsHome + CleanupConstants.REPOSITORY + File.separator + CleanupConstants.CONF +
-					File.separator + CleanupConstants.CLEANUP_PROPERTIES;
+			String configPath = bpsHome + CleanupConstants.REPOSITORY + File.separator + CleanupConstants.CONF +
+								File.separator + CleanupConstants.CLEANUP_PROPERTIES;
 			prop.load(new FileInputStream(configPath));
 		} catch (IOException e) {
-			e.printStackTrace();
+			log.error("Configuration path error.", e);
 			System.exit(0);
```
Comment (chrishantha) [inline modules/samples/product/process-tools/src/main/java/org/wso2/bps/samples/processcleanup/DBQuery.java]: Why do you exit here? If you exit due to an error, do not use 0. Try System.exit(1);

File: modules/integration/tests-integration/bpel/src/test/java/org/wso2/bps/integration/tests/bpel/security/SecurityWithServiceDescriptorTest.java (hunk 1 above)
Comment (isurusuranga) [inline modules/integration/tests-integration/bpel/src/test/java/org/wso2/bps/integration/tests/bpel/security/SecurityWithServiceDescriptorTest.java]: Fixed inhttps://github.com/wso2/product-bps/pull/100

File: modules/samples/product/process-tools/src/main/java/org/wso2/bps/samples/processcleanup/DBQuery.java (hunk 2 above)
Comment (isurusuranga) [inline modules/samples/product/process-tools/src/main/java/org/wso2/bps/samples/processcleanup/DBQuery.java]: Beacuse this is a console application which provides for the users to clean/remove the retired processes. Therefore users must have to interact with the console to delete the selected retired processes.

**Pull request merged**